README.md

```diff
@@ -72,7 +72,27 @@ introduction on how credential branding actually works.
 - [Setting up VCI frontend](./documents/vci-front-end.md)
 - [Credential Branding](./documents/credential-branding.md)
 
-### Docker
+### Docker 
+
+### Windows 10 (WSL2) - Clone repository
```

Comment:
since there's no problem with linux, isn't it better to make it a note that if you're using windows, you have to configure your git...